postgres/route.go

```diff
@@ -40,7 +40,58 @@ CREATE TABLE IF NOT EXISTS routes_stops (
 	stop_id integer REFERENCES stops NOT NULL,
 	"order" integer NOT NULL,
 	UNIQUE (route_id, "order")
-);`
+);
+CREATE TABLE IF NOT EXISTS route_schedules (
+	id serial PRIMARY KEY,
+	route_id integer REFERENCES routes ON DELETE CASCADE NOT NULL,
+	start_day smallint NOT NULL CHECK (start_day >= 0 AND start_day < 7),
+	start_time time with time zone NOT NULL,
+	end_day smallint NOT NULL CHECK (end_day >= 0 AND end_day < 7),
```

Comment:
Per discussion, the route scheduling admin interface should include a message that a schedule should not wrap around (or it could transparently split it up) since that is where admins will be modifying schedules.